azavea/docker-postgis
# Migrate from Travis to GitHub Actions

## Overview

Ports Travis CI config to a GitHub Actions workflow.

Closes #24 

### Notes

The `QUAY_IMAGES_USER` and `QUAY_IMAGES_PASSWORD` secrets have been made available to this repository.

Push and pull request Travis builds have been disabled for this repo and the Travis notify webhook has been deleted.

## Testing Instructions

 * Verify that the [GitHub Actions CI workflow](https://github.com/azavea/docker-postgis/actions/runs/181648607) passes.
* Verify that [the `cipublish` step ran](https://github.com/azavea/docker-postgis/actions/runs/181629525) ran. The `docker login` step is being used to test connectivity to Quay.io; the `docker push` command is disabled for this run and should work as long as `docker login` works.
